gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
-               .pipe(gulp.dest(dirs.dist + '/css'));
-
+                .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
Even if we want to drop support for Opera 12 because it’s an old version, there’s still Opera Mini to consider. (Btw, that’s the main reason AutoPrefixer still supports Opera 12 by default.)